mmhuman3d/data/data_structures/smc_reader.py

```diff
@@ -942,36 +973,44 @@ def get_smpl(self,
 
         # return SMPL parameters in device coordinate system
         else:
+
+            assert isinstance(self.body_model, torch.nn.Module), \
+                'body_model should be nn.Module or dict to allow rigid ' \
+                'transformation to device frame.'
+            torch_device = self.body_model.global_orient.device
+
+            assert device in {
+                'Kinect', 'iPhone'
+            }, f'Undefined device: {device}, should be "Kinect" or "iPhone"'
+            assert device_id >= 0
+
             if device == 'Kinect':
-                cam2world = self.get_kinect_color_extrinsics(
+                T_cam2world = self.get_kinect_color_extrinsics(
```

Comment:
can we rename it to trans_cam2world?

Reply:
`T_cam2world` is a (4, 4) transformation matrix, I think keeping the current name may be clearer as `trans_cam2world` may be understood as a (3, ) vector.